### PR TITLE
Move meeds code

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: _functions.sh [hunk 1]
```diff
@@ -106,6 +106,7 @@ Environment Variables
     exogtn         GateIn eXo edition                      - Apache Tomcat bundle
     plf            eXo Platform Standard Edition           - Apache Tomcat bundle
     plfcom         eXo Platform Community Edition          - Apache Tomcat bundle
+    meeds          Meeds.io                                - Apache Tomcat bundle
```
Comment (hbenali) [inline _functions.sh]: IMO, we need to specify which default docker images versions (Onlyoffice, Mysql, Postgres,...) for Meeds edition starting with 1.0 version. See https://github.com/exoplatform/adt/blob/d6c27cdab89d17625f79e0fbf6538c31b0df11a3/_functions.sh#L707
Here a suggestion: 
`      if [[ "${PRODUCT_NAME}" =~ ^(meeds) ]]; then 
           ...........
        if [[ "${PRODUCT_VERSION}" =~ ^(1.0) ]]; then
              env_var "DEPLOYMENT_FORCE_JDBC_DRIVER_ADDON" "true"
              env_var "DEPLOYMENT_ES_IMAGE_VERSION" "1.2.2"
              ...
              configurable_env_var "DEPLOYMENT_ONLYOFFICE_IMAGE_VERSION" "5.4.2.46" 
        else 
              echo_error "Product version \"${PRODUCT_VERSION}\" not yet managed"
              exit 1
        fi`

Comment (boubaker) [reply _functions.sh]: ok, I let you make the modifications onPR @hbenali , thanks !

File: var/www/lib/functions.php [hunk 2]
```diff
@@ -378,6 +429,8 @@ function getLocalAcceptanceInstances()
         case 'plfenteap':
           switch ($descriptor_array['PRODUCT_BRANCH']) {
             case '6.0.x':
+              $descriptor_array['PRODUCT_ADDONS_DISTRIB']="exo-digital-workplace / exo-es-embedded / exo-gamification / exo-kudos / exo-perk-store / exo-push-notifications / exo-wallet / exo-jcr / exo-calendar / exo-ecms / exo-tasks / exo-web-conferencing";
```
Comment (boubaker) [inline var/www/lib/functions.php]: Meeds addons ids has to change

File: var/www/lib/functions.php [hunk 3]
```diff
@@ -354,6 +403,8 @@ function getLocalAcceptanceInstances()
         case 'plfcom':
           switch ($descriptor_array['PRODUCT_BRANCH']) {
             case '6.0.x':
+              $descriptor_array['PRODUCT_ADDONS_DISTRIB']="exo-digital-workplace / exo-es-embedded / exo-gamification / exo-kudos / exo-perk-store / exo-push-notifications / exo-wallet";
```
Comment (boubaker) [inline var/www/lib/functions.php]: Meeds addons ids has to change

File: var/www/lib/functions.php (hunk 2 above)
Comment (hbenali) [reply var/www/lib/functions.php]: Done

File: var/www/lib/functions.php (hunk 3 above)
Comment (hbenali) [reply var/www/lib/functions.php]: Done

File: var/www/lib/functions.php (hunk 2 above)
Comment (boubaker) [reply var/www/lib/functions.php]: Thank you Houssem :)

**Comment by boubaker** [comment]
@vsellier could you please review this PR to unblock Meeds FT and to let us update ACC server with newer development. Thanks.

**Pull request merged**